main/src/init/factory.hpp

```diff
@@ -54,13 +56,23 @@ std::unique_ptr<ISimInitializer<Dataset>> initializerFactory(std::string testCas
     if (testCase == "sedov")
     {
         if (glassBlock.empty()) { return std::make_unique<SedovGrid<Dataset>>(); }
+#ifdef SPH_EXA_HAVE_H5PART
```

Comment:
Why is the ifdef still mentioning H5PART instead of H5 or HDF5?

Reply:
Ok. I see. To get rid of this you will need to do too many changes in the code. Not worthy.